<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editPlanOper</class>
 <widget class="QDialog" name="editPlanOper">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>319</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>From Account:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Day:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Month:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Year:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="monthBox"/>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="descrEdit"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>To Account:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="AccountComboBox" name="fromBox"/>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="yearBox"/>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="dayBox"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Summ:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="AccountComboBox" name="toBox"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Description:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Agent:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="AgentComboBox" name="agentBox"/>
     </item>
     <item row="6" column="1">
      <widget class="currencyEdit" name="curr_Edit"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>3</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="OkPushButton" name="okButton">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CancelPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>OkPushButton</class>
   <extends>QPushButton</extends>
   <header location="global">widgets/okpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>AccountComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">widgets/accountcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>currencyEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">widgets/currencyedit.h</header>
  </customwidget>
  <customwidget>
   <class>CancelPushButton</class>
   <extends>QPushButton</extends>
   <header location="global">widgets/cancelpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>AgentComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">widgets/agentcombobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>dayBox</tabstop>
  <tabstop>monthBox</tabstop>
  <tabstop>yearBox</tabstop>
  <tabstop>fromBox</tabstop>
  <tabstop>agentBox</tabstop>
  <tabstop>toBox</tabstop>
  <tabstop>curr_Edit</tabstop>
  <tabstop>descrEdit</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
